### PR TITLE
Fix: Travis build runs tests twice

### PR DESCRIPTION
**Pull request by cythrawll**
## Description

travis running tests twice, this doesn't seem to be the intention

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ before_install:
 install:
   - travis_wait npm install
```
Comment (kevinhinterlong) [inline .travis.yml]: Maybe switch this over to [`npm ci`](https://docs.npmjs.com/cli/ci.html) while we're at it

Comment (cythrawll) [reply .travis.yml]: sure did that, unfortunately`lerna bootstrap` won't use `npm ci` until we upgrade lerna to `3.x` we had problems with upgrading however, we should look into it again.

Comment (balajimanoharan) [reply .travis.yml]: lerna publish in the latest version fails with the current way we publish. Either we would need a tag in npm to use the current publish method or swap our current publish method to not do a canary release.

Comment (cythrawll) [reply .travis.yml]: or we can release a version, that would work too

**Pull request merged**